en/airframes/airframe_reference.md

```diff
@@ -39,6 +39,34 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Dodecarotor cox
```

Comment:
@bkueng I mean a rotor-order diagram as per all the other copters in the airframe docs. As far as I can tell, not added in that PR. It is possible I am being dim!